src/main/integrations/discord-presence/index.ts

```diff
@@ -1,14 +1,16 @@
-import playerStateStore, { PlayerState, Thumbnail, VideoState } from "../../player-state-store";
+import playerStateStore, { PlayerState, Thumbnail, VideoState, VideoDetails } from "../../player-state-store";
 import IIntegration from "../integration";
+import Conf from "conf";
 import MemoryStore from "../../memory-store";
-import { MemoryStoreSchema } from "~shared/store/schema";
+import { MemoryStoreSchema, StoreSchema } from "~shared/store/schema";
+import { Unsubscribe } from "conf/dist/source/types";
 import DiscordClient from "./minimal-discord-client";
 import log from "electron-log";
 import { DiscordActivityType } from "./minimal-discord-client/types";
 
 const DISCORD_CLIENT_ID = "1143202598460076053";
 
-function getHighestResThumbnail(thumbnails: Thumbnail[]) {
+function getBestThumbnail(thumbnails: Thumbnail[]): string {
```

Comment:
I think the original function name is better suited as it's describing exactly what the function is doing.

Reply:
Please do not resolve this without at least commenting?

Reply:
I changed the name of the function back

Reply:
https://github.com/Akiisqt/ytmdesktop/commit/6439a53aaa0178a610091fae90a2efbea166d91a - You fixed it
https://github.com/Akiisqt/ytmdesktop/commit/81cbd1e75b0cd45e6a5edb6a40d880717fb4370e - You reversed it back to the **getBestThumbnail** function... 😖

Reply:
see that makes sense now